### ISSUE TITLE
May I ask if it supports highlighting in formats like 0xffaaaaaaa

### ISSUE THREAD

**Issue by reimu1234** [brenoprata10/nvim-highlight-colors#67]
A very beautiful project, thank you for your work. 
I found that the format 0xffaadd is acceptable when using it, but 0xaarrggbb does not work
May I ask if it supports highlighting in formats like 0xffaaaaaaa thank you

**Comment by brenoprata10** [comment]
What is the difference of `0xaarrggbb` and `0xffaadd`? Are they the same thing, but with additional configuration like opacity?
I am not familiar with this color code format, so I would appreciate if you could explain it to me 😜

**Comment by yzwik** [comment]
Yes, it's just that there's more transparency, there's no other difference

**Comment by bassamsdata** [comment]
supporting this format would be awesome.

It's an Android color format. also used in some projects I use like [sketchybar](https://github.com/FelixKratz/SketchyBar). here's a [color picker](https://felixkratz.github.io/SketchyBar/config/tricks#color-picker) for it as well.

in addition, for more info here is a simple [stackoverflow](https://stackoverflow.com/questions/10788858/how-to-convert-android-color-format) answer.

As a side note, nvim-colorizor supports this format as well, so might benefit from it's integration, although I know this plugin is totally different.

at the end, I'd like to thank you for this cool plugin.

**Comment by yzwik** [comment]
> supporting this format would be awesome.
> 
> It's an Android color format. also used in some projects I use like [sketchybar](https://github.com/FelixKratz/SketchyBar). here's a [color picker](https://felixkratz.github.io/SketchyBar/config/tricks#color-picker) for it as well.
> 
> in addition, for more info here is a simple [stackoverflow](https://stackoverflow.com/questions/10788858/how-to-convert-android-color-format) answer.
> 
> As a side note, nvim-colorizor supports this format as well, so might benefit from it's integration, although I know this plugin is totally different.
> 
> at the end, I'd like to thank you for this cool plugin.

but nvim-colorizor it seems that the format 0xrrggbb is not supported

**Comment by bassamsdata** [comment]
> but nvim-colorizor it seems that the format 0xrrggbb is not supported

True, but this plugin does support it. I think the issue is with the format `0xAARRGGBB`, isn't it?

**Comment by yzwik** [comment]
> > but nvim-colorizor it seems that the format 0xrrggbb is not supported
> 
> True, but this plugin does support it. I think the issue is with the format `0xAARRGGBB`, isn't it?

yes

**Comment by RolfSievert** [comment]
Just a suggestion; could also make it possible to add custom formats. What I mean is that in the config you could add:
```lua
custom_formats = {
    '0xAARRGGBB',
    '0xRRGGBBAA',
    -- etc
}
```
, where `A`, `R`, `G`, and `B` are reserved characters for matching color channels.

**Comment by nickvonkaenel** [comment]
Loving the plugin, I would appreciate support for 0xRRGGBBAA as well.
